src/operun/crm/setuphandlers.py

```diff
@@ -16,8 +17,18 @@ def getNonInstallableProfiles(self):
 def post_install(context):
     """Post install script"""
     # Do something at the end of the installation of this package.
+    _displayed_types()
 
 
 def uninstall(context):
     """Uninstall script"""
     # Do something at the end of the uninstallation of this package.
+
+
+def _displayed_types():
+    """
+    Add Training to displayed types.
+    """
+    types = api.portal.get_registry_record('plone.displayed_types')
+    types = types + (u'Accounts', u'Contacts', u'Todos',  u'Files')
```

Comment:
On existing Database the old content type name is in the field. During the migration we have to replace custom content type names in this field. For now i'd override with Plone default + our custom types.